pluginLauncher/lib/include/DobbyRdkPluginUtils.h

```diff
@@ -39,15 +39,28 @@
 #include <memory>
 #include <list>
 #include <mutex>
+#include <arpa/inet.h>
+
 
 // TODO:: This would be better stored in the dobby workspace dir rather than /tmp,
 // but we don't programatically know the workspace dir in this code.
-#define ADDRESS_FILE_PREFIX       "/tmp/dobbyAddress_"
+#define ADDRESS_FILE_DIR          "/tmp/dobby/plugin/networking/"
 
 typedef struct ContainerNetworkInfo
 {
     std::string vethName;
     std::string ipAddress;
+    in_addr_t ipAddressRaw;
+    std::string containerId;
+
+    bool operator==(const ContainerNetworkInfo &rhs) const
+    {
+        if (containerId.empty())
```

Comment:
The condition should probably be:
if (containerId.empty() || rhs.containerId.empty())
because as for now there are situations where A==B and B==A could have different results.